.github/workflows/daily_update.yml

```diff
@@ -1,5 +1,8 @@
 name: Daily Update from Dark Visitors
 on:
+  push:
```

Comment:
I think this file needs renaming now that it is more than just daily updates.

There could be a small downside to combining the function in this way: updates on merge could fail if the dark visitors site is down. But I like doing the dark visitors merge in more cases, e.g. if we change the dark visitors handling code, we'd want to re-run it on merge. So, on balance, it's probably a good approach.

Reply:
A better name would be useful. What should be though? `ai_robots_update`?

I did not consider this, but will address this in this PR.

Reply:
Let's think about this. The inputs are `robots.json` and information from the dark visitors site and the outputs are `robots.txt` and `table-of-bot-metrics.md`. In other words, it's complicated. So I think some generic name like "generate" or "update" might be enough, given the context.

Reply:
Oh and the action name should use the same terminology as the filename, but can be a bit wordier.

Reply:
The actions are quite fast. So they can easily run on each push by my judgement.

I did add some error handling for the case the script is run and cannot connect to darkvisitors.com.